src/pspm_load_channel.m

```diff
@@ -0,0 +1,104 @@
+function [sts, data_struct, infos] = pspm_load_channel(fn, channel, channeltype)
```

Comment:
May I confirm the output arguments should be `sts` and `data_struct`, or the three arguments? The *format* written below is different from this line.

Reply:
Thanks - I adapted the help text.